src/exporters/insomnia/types/request.ts

```diff
@@ -3,14 +3,14 @@ import RequestBody from './request_body';
 class Request {
     _type: string = "request";
     _id: string;
-    parentId: string;
     created: number;
-    modified: number;
     name: string;
-    method: string = "POST";
-    url: string;
-    body: RequestBody;
+    method: HTTPMethod = "POST";
     headers: any[] = [];
+    parentId: Maybe<string>;
```

Comment:
?What's the implication of using `Maybe<string>` as against `string`?

Reply:
In this case is a hand-made type (I created it [here](https://github.com/oshosanya/graphql-export/pull/10/files#diff-af565283322a1965b4ea9bfb48fe1f578631a934aadbfc7a413fe0d5e84746fbR1)). But the _concept_ comes from functional programming:
>In programming languages (more so functional programming languages) and type theory, an option type or maybe type is a polymorphic type that represents encapsulation of an optional value; e.g., it is used as the return type of functions which may or may not return a meaningful value when they are applied. It consists of a constructor which either is empty (often named `None` or `Nothing`), or which encapsulates the original data type `A` (often written `Just A` or `Some A`).

[Wikipedia reference](https://en.wikipedia.org/wiki/Option_type)

----

The way that I've written it, it means: "the value is either a `string` (in this case) or `Nothing` (represented as `null | undefined`)

Reply:
I chose using this Maybe as the "path of least resistance". In reality, I think some of these fields could be completly removed (i.e. they never have values assigned)

Reply:
I get this now, makes a lot of sense